webview/__init__.py

```diff
@@ -255,6 +256,7 @@ def create_window(
     server: type[http.ServerType] = http.BottleServer,
     http_port: int | None = None,
     server_args: http.ServerArgs = {},
+    event_loop: AbstractEventLoop = None
```

Comment:
I suggest moving event_loop as a parameter of the `start` function instead. Having a window-dedicated event loop  is a bit overkill imo.